### DISCUSSION TITLE
The new behavior of the fire extinguisher bracket and weapon holder is inconvenient.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14425] [Ideas and suggestions]
The fact that the fire extinguisher can be removed from the wall with one click of a button is inconvenient and leads to undesirable consequences.
For example, at the station, if you accidentally click on the fire extinguisher, it is easy to remove it from the wall and get a reputation penalty.
During an intense battle in a submarine, it is easy to miss with the mouse and end up with a fire extinguisher in your hands. In the previous version of the game, when you click on the fire extinguisher bracket or the weapon holder , a window opened. This was convenient.

**Comment by SomeRandomNoobKekeke** [comment]
Actually rep loss is a fraction, and i believe you need to steal 4 extinguishers to get full -1.0 rep

**Comment by pavel4win** [comment]
Unfortunately, this is not the case. Go to the fire extinguisher yourself and take it in front of the station staff, you will immediately get -1 to your reputation, even if you immediately dropped it. Fire extinguishers are often hanging next to doors, so it is often possible to grab a fire extinguisher by missing the button that opens the door. It would be better if it were like before, the first click opens the fire extinguisher container and after that you can take the fire extinguisher. This new behavior of the fire extinguisher sometimes gets in the way on your submarine. In the heat of battle, having missed the right button, you find that your hands are busy with the fire extinguisher and you lose time to drop it. It's good that now there is a hot key to drop the item.

**Reply by SomeRandomNoobKekeke** [reply]
I mean first one gives you -1, next 3 is free

**Reply by pavel4win** [reply]
It calmed me down :)

Lets make it so that only taking the fire extinguisher out of the station reduces reputation.
Or give at least 10 seconds to drop it.
Or better return back the first click opens the container.
This applies, by the way, to weapon's & backpack's holder.
And IMHO weapon's & backpack's holder should fit a watering can.
